src/components/badge/themes/commerce.json

```diff
@@ -0,0 +1,57 @@
+{
+	"badge": {
+		"styles": {
+			"default": {
+				"align-self": "baseline",
+				"border-radius": "var(--border-radius-pill)",
+				"color": "var(--global-white)",
+				"font-size": "var(--body-font-size-tiny)",
+				"font-weight": "var(--heading-level-5-font-weight)",
```

Comment:
@nschubach you were curious as well what differences I surfaced between the news and commerce sass map from a values standpoint. Here's an example.

And in the news.json it's `"font-weight": "var(--global-font-weight-7)",`